interactions/client/bot.py

```diff
@@ -363,6 +365,55 @@ async def change_presence(self, presence: ClientPresence) -> None:
         """
         await self._websocket._update_presence(presence)
 
+    async def get(
```

Comment:
I don't get the purpose of adding the `get()` method when there is no way to demand data out of our cache.

Reply:
It still simplifies getting models from the API for the end user.

Reply:
Not sure why you resolved this conversation 😅 I'll be re-opening it to continue discussion on this part.

Grabbing from cache is not a suggestion, it's an absolute requirement of the `get()` method. We want this because we want to make as few HTTP calls as possible, which could easily help us hit rate limits depending on how bot developers structure their callbacks for this method. The reason why we've been postponing this for so long is because cache needs a refactor in not only the way we're storing data, but the way we *update* the cache itself. In theory, the future of this library will involve us heavily interacting with the cache and making rare HTTP requests unless absolutely needed. Because of this, it needs to be possible for a bot developer to save themselves the hassle of potential rate limits by reading from the cache first before making an HTTP request. We do this especially with our application command synchronisation process to avoid excessive HTTP requests leading to rate limits, an effective measure that's been tried-and-tested.

Reply:
Yes I understand, I just haven't utilized the cache since it needs a refactor.
I will keep this as a draft until 4.1.1 is released and caching is implemented.
I want to continue work on your suggestions till then.